pkg/controller/toolchainstatus/toolchainstatus_controller.go

```diff
@@ -351,6 +354,10 @@ func (r *ReconcileToolchainStatus) membersHandleStatus(reqLogger logr.Logger, to
 func (r *ReconcileToolchainStatus) sendToolchainStatusUnreadyNotification(logger logr.Logger,
 	toolchainStatus *toolchainv1alpha1.ToolchainStatus) error {
 
+	if !isValidEmailAddress(r.config.GetAdminEmail()) {
+		return errs.New("cannot create notification due to configuration error - admin.email is invalid or not set")
```

Comment:
Let's add the problematic email to the error message

Reply:
Done